ui/pages/onboarding-flow/recovery-phrase/recovery-phrase-chips.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import RecoveryPhraseChips from './recovery-phrase-chips';
+
+describe('Recovery Phrase Chips Component', () => {
+  const TEST_SEED =
+    'debris dizzy just program just float decrease vacant alarm reduce speak stadium';
+
+  it('should match snapshot', () => {
+    const props = {
+      secretRecoveryPhrase: TEST_SEED.split(' '),
+    };
+
+    const { container } = renderWithProvider(
+      <RecoveryPhraseChips {...props} />,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
```

Comment:
I think if we test snapshot for `hideSeedPhrase` as `true` and `false`, will cover more situations. wdyt